leftwm/src/bin/leftwm.rs

```diff
@@ -159,10 +170,14 @@ fn start_leftwm() {
             std::thread::sleep(delay);
         }
     }
+
+    if !no_error_occured {
```

Comment:
follow up from above:
```suggestion
    if error_occured {
```